lib/lower/PFTBuilder.cpp

```diff
@@ -507,13 +531,32 @@ void dumpFunctionLikeUnit(llvm::raw_ostream &outputStream,
       },
       functionLikeUnit.funStmts.front());
   outputStream << unitKind << ' ' << name;
+  dumpParentInfo(outputStream, functionLikeUnit);
   if (header.size())
     outputStream << ": " << header;
   outputStream << '\n';
   dumpEvalList(outputStream, functionLikeUnit.evals);
+  if (!functionLikeUnit.funcs.empty()) {
+    outputStream << "\nContains\n";
+    for (auto &func : functionLikeUnit.funcs) {
+      dumpFunctionLikeUnit(outputStream, func);
+    }
```

Comment:
LLVM coding standard difference here.

Reply:
Ah, you mean I should get rid of the `{ }` ? OK, that is the one LLVM rule that always bites me when later extending the loops, but hey, if that is the rule :)